Formula/minimesos.rb

```diff
@@ -0,0 +1,14 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://raw.githubusercontent.com/ContainerSolutions/minimesos/0.10.2/bin/minimesos"
+  sha256 "58d05e2574e1c82b8f2f366f92d6dc464438603bd0fb5fb4b6383b6123233af0"
+
+  def install
+    bin.install "minimesos"
+  end
+
+  test do
+    system "minimesos", "version"
```

Comment:
Test is failing, seems to need `depends_on "docker"`: https://bot.brew.sh/job/Homebrew%20Core/9364/version=any_sierra/testReport/junit/brew-test-bot/sierra/test_minimesos/

Also, use `system "#{bin}/minimesos"`